README.md

```diff
@@ -96,7 +96,8 @@ spec:
 
 ### NOTES
 - Reloader also supports [sealed-secrets](https://github.com/bitnami-labs/sealed-secrets). [Here](docs/Reloader-with-Sealed-Secrets.md) are the steps to use sealed-secrets with reloader.
-- `reloader.stakater.com/auto: "true"` will always override when use with either `secret.reloader.stakater.com/reload` or `configmap.reloader.stakater.com/reload` annotation.
+- `reloader.stakater.com/auto: "true"` will only reload the pod, if the configmap or secret is used (as a volume mount or as an env) in `Deployment/Daemonsets/Statefulsets`
+- `secret.reloader.stakater.com/reload` or `configmap.reloader.stakater.com/reload` annotation will reload the pod upon changes in specified configmap or secret, irrespective of the usage of configmap or secret.
 - you may override the auto annotation with the `--auto-annotation` flag
```

Comment:
@faizanahmad055 can't we have just one annotation? e.g. `always-reload` or `force-reload` why do we need two annotations? We should keep it as simple as possible and as easy as possible

Reply:
We have three annotations at the moment
1. reloader.stakater.com/auto
2. secret.reloader.stakater.com/reload
3. configmap.reloader.stakater.com/reload

**reloader.stakater.com/auto:**
We need this annotation to automatically reload the pod if mounted configmap or secret changes. This is the most easiest way.

**secret.reloader.stakater.com/reload & configmap.reloader.stakater.com/reload:**
These caters to specific scenarios, for example,
1.  If a pod depends on multiple configmaps, but the user wants to reload the pod for only one configmap change, then one of these annotations is used.
2. If a pod depends on a configmap but the configmap is not used in deployment (i.e. mounted or used as an env), then we need one of these annotations to reload the pod upon change in config map.

To explain why we need both of these annotations `secret.reloader.stakater.com/reload & configmap.reloader.stakater.com/reload` and why don't we use one general purpose annotation is because secret and configmap can have same name. And separate annotations for secret and configmap help reloader to determine whether should it reload for a secret or a configmap if both has same name.